scripts/serialize_new_errata_rec.py

```diff
@@ -0,0 +1,139 @@
+
+    data = {}
+    for column in inspect(instance).mapper.column_attrs:
+        value = getattr(instance, column.key)
+
+        if isinstance(value, Enum):
+            data[column.key] = value.name
+        elif isinstance(value, datetime.datetime):
+            data[column.key] = value.isoformat()
+        elif isinstance(column.expression.type, JSONB):
+            data[column.key] = json.dumps(value)
+        else:
+            data[column.key] = value
+
+    for relationship in inspect(instance).mapper.relationships:
+        related_value = getattr(instance, relationship.key)
+        if related_value is None:
+            data[relationship.key] = None
+        elif relationship.uselist:
+            if type(related_value[0]) in exclude_models:
+                data[relationship.key] = []
+                continue
+            data[relationship.key] = [
+                serialize_model(item, seen, exclude_models)
+                for item in related_value
+            ]
+        else:
+            data[relationship.key] = serialize_model(
+                related_value, seen, exclude_models
+            )
+
+    return data
+
+
+def deserialize_model(cls, data):
+    instance = cls()
+
+    if data is None:
+        return instance
+
+    for column in inspect(cls).columns:
+        value = data.get(column.key)
+
+        if value is None:
+            continue
+
+        if isinstance(column.type, sqlalchemy.Enum):
+            enum_class = column.type.python_type
+            value = enum_class[value]
+
+        elif isinstance(column.type, sqlalchemy.DateTime):
+            value = datetime.datetime.fromisoformat(value)
+
+        elif isinstance(column.type, JSONB):
+            value = json.loads(value) if isinstance(value, str) else value
+
+        setattr(instance, column.key, value)
+
+    for relationship in inspect(cls).relationships:
+        related_data = data.get(relationship.key)
+        if related_data is not None:
+            if isinstance(related_data, dict):
+                related_instance = deserialize_model(
+                    relationship.mapper.class_, related_data
+                )
+                setattr(instance, relationship.key, related_instance)
+
+            elif isinstance(related_data, list):
+                related_instances = [
+                    deserialize_model(relationship.mapper.class_, item)
+                    for item in related_data
+                ]
+                setattr(instance, relationship.key, related_instances)
+
+    return instance
+
+
+if __name__ == '__main__':
+    logging.basicConfig(level=logging.INFO, handlers=[logging.StreamHandler()])
+
+    sqlalchemy_url = os.getenv('SQLALCHEMY_URL')
+
+    engine = create_engine(sqlalchemy_url, echo=False)
+    session = Session(engine)
+    stmt = select(NewErrataRecord).where(
+        NewErrataRecord.id.in_((
+            'ALSA-2022:4991',
+            'ALSA-2022:4887',
```

Comment:
This one is not released and won't be processed (in fact, is not present in the xml sample you provided), so it doesn't really bring any value here.

I'd recommend to use modern and released ones, here's a list:
* ALSA-2025:0281 (single pkg)
* ALSA-2025:0325 (multiple pkgs | including noarch pkgs)
* ALSA-2024:9644 (module | multiple pkgs)
* ALSA-2024:6964 (module | devel module | multiple pkgs)